gradle.properties

```diff
@@ -22,4 +22,7 @@ kotlin.code.style=official
 
 RELEASE_REPOSITORY_URL https://nexus.iroha.tech/repository/maven-soramitsu/
 SNAPSHOT_REPOSITORY_URL https://nexus.iroha.tech/repository/maven-soramitsu/
+android.defaults.buildfeatures.buildconfig=true
+android.nonTransitiveRClass=false
+android.nonFinalResIds=false
```

Comment:
I think that `android.nonFinalResIds=true` should work correctly. Why do we need to keep the old version of `android.nonFinalResIds`?